packages/cli/src/formatter.ts

```diff
@@ -326,6 +327,47 @@ export const formatDeployActions = ({
   ]
 }
 
+export const formatDeploymentSummaryResult = (ids: DetailedChangeId[], result: DeploySummaryResult): string => {
+  const mapping: Record<DeploySummaryResult, DetailedChangeId | null> = {
+    failure: Prompts.DEPLOYMENT_STATUS.failure,
+    'partial-success': Prompts.DEPLOYMENT_STATUS.partialSuccess,
+    success: Prompts.DEPLOYMENT_STATUS.success,
+  }
+
+  const sign: string | null = mapping[result]
```

Comment:
```suggestion
  const sign = mapping[result]
```
you dont need to state the types everywhere, typescript know most of the types by itself. so Im not commenting on all of them but remove them